.circleci/config.yml

```diff
@@ -227,10 +230,28 @@ jobs:
           name: Run e2e file upload features test
           command: |
             python -m scripts.run_e2e_tests --skip-install --suite="fileUploadFeatures"
+      - run:
+          name: Run e2e play voiceovers test
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="playVoiceovers"
+      - run:
+          name: Run e2e file upload extensions test
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="fileUploadExtensions"
+
+  e2e_non_prod_new_structure_tests:
+    <<: *job_defaults
+    steps:
+      - checkout
+      - check_release_chaneglog_update_branch
```

Comment:
typo.

Reply:
Fixed it throughout teh file.

Also, FYI the new_strcuture test again failed twice consecutively on this with flaky errors.